packages/generator/templates/app/app/blitz-client.ts

```diff
@@ -0,0 +1,12 @@
+import {AuthClientPlugin} from "@blitzjs/auth"
+import {setupClient} from "@blitzjs/next"
+
+const {withBlitz} = setupClient({
+  plugins: [
+    AuthClientPlugin({
+      cookiePrefix: "webapp-cookie-prefix",
```

Comment:
We should have cookiePrefix: "__safeNameSlug__", which then will be replaced by the genrator

Reply:
After RPC client plugin PR is merged, we also need to pass the RPC plugin